pkg/machines/libvirt-common.es6

```diff
@@ -566,6 +579,23 @@ export function updateVCPUSettings(domXml, count, max, sockets, cores, threads)
     return tmp.innerHTML;
 }
 
+export function updateMaxMem(domXml, maxMemory) {
+    const domainElem = getDomainElem(domXml);
+
+    let maxMemElem = domainElem.getElementsByTagName("maxMemory")[0];
+    if (maxMemElem) {
+        maxMemElem.textContent = `${maxMemory}`;
+    } else {
+        let memElem = domainElem.getElementsByTagName("memory")[0];
+        memElem.textContent = `${maxMemory}`;
+    }
+
+    const tmp = document.createElement("div");
+    tmp.appendChild(domainElem);
+
+    return tmp.innerHTML;
+}
+
```

Comment:
Please use the libvirt APIs for setting memory and max memory and not build the xml and redefine the domain.
https://libvirt.org/html/libvirt-libvirt-domain.html#virDomainSetMemoryFlags

Reply:
SetMemory flags will only work on shut off domains as discussed, so this comment is invalid.